client/layout/store-notices/index.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { compose } from '@wordpress/compose';
+import PropTypes from 'prop-types';
+import { Notice } from '@wordpress/components';
+
+/**
+ * WooCommerce dependencies
+ */
+import { OPTIONS_STORE_NAME } from '@woocommerce/data';
+
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import withSelect from 'wc-api/with-select';
+
+class StoreWarnings extends Component {
+	isVisibleMissingPaimentWarning() {
+		const { paymentMethods, queryString } = this.props;
+		const isPaymentPage = queryString.task
+			? queryString.task === 'payments'
+			: false;
+		return ! Object.keys( paymentMethods ).length && isPaymentPage;
+	}
+
+	render() {
+		const showMissingPaimentMethod = this.isVisibleMissingPaimentWarning();
+
+		return (
+			showMissingPaimentMethod && (
+				<Notice
+					className="woocommerce-store-warnings"
+					status="warning"
+					isDismissible={ false }
+				>
+					<p>
+						Customers cannot place orders at your store until you
```

Comment:
This string needs to be translated.

Reply:
Hi @timmyc, thank you for your review.
Nice catch! I added it in the commit d9c0a3d